ansible/collections/bento/common/roles/build_frontend/files/inject.template.js

```diff
@@ -15,4 +15,5 @@ window.injectedEnv = {
     REACT_APP_USER_SERVICE_API: '${REACT_APP_USER_SERVICE_API}',
     REACT_APP_NIH_CLIENT_ID: '${REACT_APP_NIH_CLIENT_ID}',
     REACT_APP_NIH_AUTH_URL: '${REACT_APP_NIH_AUTH_URL}',
+    REACT_APP_BACKEND_PUBLIC_API: '${REACT_APP_BACKEND_PUBLIC_API}'
```

Comment:
Codacy has a fix for the issue: [Replace `····REACT_APP_BACKEND_PUBLIC_API:·'${REACT_APP_BACKEND_PUBLIC_API}'` with `··REACT_APP_BACKEND_PUBLIC_API:·"${REACT_APP_BACKEND_PUBLIC_API}",`](https://app.codacy.com/gh/CBIIT/datacommons-devops/pullRequest?prid=10017653)
```suggestion
  REACT_APP_BACKEND_PUBLIC_API: "${REACT_APP_BACKEND_PUBLIC_API}",
```